src/arch/arm/64/kernel/vspace.c

```diff
@@ -1172,8 +1172,12 @@ static exception_t performPageInvocationUnmap(cap_t cap, cte_t *ctSlot)
                   cap_frame_cap_get_capFBasePtr(cap));
     }
 
-    cap_frame_cap_ptr_set_capFMappedASID(&ctSlot->cap, asidInvalid);
-    cap_frame_cap_ptr_set_capFMappedAddress(&ctSlot->cap, 0);
+    cap_t slotCap = ctSlot->cap;
```

Comment:
Can't the variable name be simply `cap` to be consistent with the naming?

Reply:
Indeed. I copied it from RISCV so that the proof would be identical, did not notice that page inv. unmap and asid pool invocations use different names for the var. Now the question is whether to be consistent with RISCV or with the other function, as we don't have time to update RISCV right this second. Thoughts welcome.

Reply:
I don't know enough about the proof side and the impact there. Are parts of the proofs shared? I consider the name "slotCap" a bit unfortunate because we have "cap slots" also. So using just "cap" makes the C code a bit easier to understand in term of what is done.

Reply:
We should do whatever makes the verification work easier.

Reply:
It's only a small impact on verification (the C refinement proofs are not shared between arches, maybe someday), so yeah, might be nicer as "cap" then in line with the other functions. I'll do that, and we can leave the RISCV64 tweak to some another day.

Reply:
Right, in that case `slotCap` it is. The name is used a lot in decode functions, it just means "the cap in `slot`".